README.md

```diff
@@ -1,7 +1,17 @@
-# TiKV
+![tikv_logo](images/logo.png)
```

Comment:
Please use a higher quality image, preferring an SVG. This is noticeably pixelated on high DPI screens.